doc/options.jax

```diff
@@ -400,6 +410,30 @@ Note: 将来的に、より多くのグローバルオプションが global-loc
 ない。
 
 
+						*option-value-function*
+いくつかのオプション('completefunc', 'imactivatefunc', 'imstatusfunc',
+'omnifunc', 'operatorfunc', 'quickfixtextfunc', 'tagfunc', 'thesaurusfunc')は
+関数名もしくは関数への参照もしくはラムダ関数を設定する。ラムダを使う場合は内部
+名へ変換される、例えば "<lambda>123"。例:
+>
+	set opfunc=MyOpFunc
+	set opfunc=function('MyOpFunc')
+	set opfunc=funcref('MyOpFunc')
+	set opfunc={a\ ->\ MyOpFunc(a)}
+	" 関数への参照の変数を用いて設定
+	let Fn = function('MyTagFunc')
+	let &tagfunc = Fn
+	" ラムダ式を用いて設定
+	let &tagfunc = {t -> MyTagFunc(t)}
+	" ラムダ式を含む変数を用いて設定
+	let L = {a, b, c -> MyTagFunc(a, b , c)}
+	let &tagfunc = L
+
+Vim9 script において、コンパイル済み関数内では、ラムダは使うことができるが、ク
+ロージャは動作しない、なぜなら関数が定義された位置のコンテキストを持たずに呼ば
+れることになるため。
```

Comment:
- `ラムダは` --> `ラムダを`
- 日本語の場合は、`なぜなら`の前で一旦「。」で切った方が良さそうです。
  または、`because～`を先に持ってきて、「Vim9  script では、コンパイル済み関数内でラムダを使用できるが、関数が定義された場所のコンテキストなしで呼び出されてしまうため、クロージャは動作しない。」等とするかですかね。

```suggestion
Vim9 script において、コンパイル済み関数内では、ラムダを使うことができるが、ク
ロージャは動作しない。なぜなら、関数が定義された位置のコンテキストを持たずに呼
ばれることになるためである。
```

Reply:
たしかに、この方がよさそうです。